### PR TITLE
Update files by generated versions

### PR DESCRIPTION
**Pull request by ybznek**
These changes were caused by generating of remediation scripts using 
- ./create_*_disabled.py + csv files
- manually written script.

Should only make easier merging/reviewing of next infrastructure changes.

### CODE REVIEW

File: shared/remediations/bash/kernel_module_bluetooth_disabled.sh [hunk 1]
```diff
@@ -1,2 +1,3 @@
-# platform = multi_platform_rhel
+# platform = multi_platform_rhel, multi_platform_fedora
+# THIS FILE IS GENERATED. DO NOT EDIT!
 echo "install bluetooth /bin/true" > /etc/modprobe.d/bluetooth.conf
```
Comment (redhatrises) [inline shared/remediations/bash/kernel_module_bluetooth_disabled.sh]: This should be updated to be idempotent. Should probably make sure that the template makes it idempotent as well.

File: shared/remediations/bash/kernel_module_usb-storage_disabled.sh [hunk 2]
```diff
@@ -1,2 +1,3 @@
 # platform = multi_platform_rhel, multi_platform_fedora
+# THIS FILE IS GENERATED. DO NOT EDIT!
 echo "install usb-storage /bin/true" > /etc/modprobe.d/usb-storage.conf
```
Comment (redhatrises) [inline shared/remediations/bash/kernel_module_usb-storage_disabled.sh]: Same as above.

Comment (ybznek) [inline shared/remediations/bash/kernel_module_usb-storage_disabled.sh]: @redhatrises Thank you for hint, but these issues are not related to this PR (see changed lines).
I have created issue assigned to me for that https://github.com/OpenSCAP/scap-security-guide/issues/1369. 

I am working on some structural changes and I cannot fix such issue during the work.

**Comment by ybznek** [comment]
test this please - some of jenkins slaves were down.

**Comment by iankko** [comment]
> @redhatrises Thank you for hint, but these issues are not related to this PR (see changed lines).
I have created issue assigned to me for that #1369.
> I am working on some structural changes and I cannot fix such issue during the work.

@redhatrises 
+1 on this approach. Though I agree the templates need rewrite, it's IMHO not good practice to add issues to PR on the way as they are created (because this way we could quickly end up in state, to get something done, we would first need to rewrite 1/2 of the repository).

It's enough when the problem is tracked (#1369) - to be fixed in the future.

**Comment by shawndwells** [comment]
agree with @iankko , the templates should be fixed independently
